deploy/cert-manager-webhook-anexia/templates/pki.yaml

```diff
@@ -0,0 +1,76 @@
+---
+# Create a selfsigned Issuer, in order to create a root CA certificate for
+# signing webhook serving certificates
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ include "cert-manager-webhook-anexia.selfSignedIssuer" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ include "cert-manager-webhook-anexia.name" . }}
+    chart: {{ include "cert-manager-webhook-anexia.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selfSigned: {}
+
+---
+
+# Generate a CA Certificate used to sign certificates for the webhook
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "cert-manager-webhook-anexia.rootCACertificate" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ include "cert-manager-webhook-anexia.name" . }}
+    chart: {{ include "cert-manager-webhook-anexia.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  secretName: {{ include "cert-manager-webhook-anexia.rootCACertificate" . }}
+  duration: 43800h # 5y
```

Comment:
not necessarily to be changed, but with this timeout we will have a lot of fun in five years if auto renew does not work as intended. I'd like a way shorter timeout - better run into problems while we remember this at least a bit than in five years. And since this is auto renewing, a short timeout isn't a problem - if everything works as intended.

Reply:
This is just the default value from upstream, maybe @HaveFun83 has an opinion here?

Reply:
I thing we should keep the upstream values here but if we want to change it we should made the duration configurable via helm values.

Reply:
I've replaced the static values with references to 

```yaml
rootCACertificateDuration: 43800h # 5y
servingCertificateDuration: 8760h # 1y
```